spec/ns-onco-core/oncocore-genetics.txt

```diff
@@ -31,27 +31,32 @@ Conformance note: To be conformant to US Core, the code attribute must be a LOIN
 Implementation note: The performer of the test (organization or practitioner) should be included in the FHIR profile as the performer.actor.
 
 Conformance note: The category for this profile is set to GE (Genetics), a code from http://hl7.org/fhir/ValueSet/diagnostic-service-sections. This is contrary to the [Argonaut](http://www.fhir.org/guides/argonaut/r2/) and [US Core](http://hl7.org/fhir/us/core/STU3/index.html) specifications, which require the category 'LAB' in diagnostic reports containing laboratory results. This is assumed to be an oversight in the US Core and Argonaut specifications."
-// Property: SpecimenType 0..1
-// Property:          RegionStudied 0..*
+
                    ObservationSubjectOfRecord only Patient
-                   // US Core v3 requires an extensible binding to Code
-//                   Code from GeneticTestVS (extensible)
                    Code = LNC#81247-9 "Master HL7 genetic variant reporting panel"
                    Specimen substitute GeneticSpecimen
-//                   Category = V20074#GE
                    Observation
-//                   includes GeneticVariantFound 0..*
                    includes GeneticVariant 0..*
                    includes RegionStudied 0..*
                    // SpecimenType from GeneticSpecimenTypeVS (extensible)
 
 
-Entry:           RegionStudied
-Parent:            CodedNonLaboratoryObservation
-Description:       "A description of the coverage of the genome that was tested for variants."
-//Value:             string
+Entry:              RegionStudied
+Parent:             CodedNonLaboratoryObservation
+Description:        "Gene mutations that were tested for."
```

Comment:
The description needs to be clarified and expanded. Is this really a _list_ of gene mutations tested for, or the area of the genome tested for variants?  I thought the previous description was accurate. What is going on here?

Reply:
Probably makes sense to make it as generic as possible since the CG WG is still debating its use. Regardless, it's meant to describe "the area of the genome tested for variants".  I'll change the description to reflect that.